resources/translations/fi.edn

```diff
@@ -137,7 +137,7 @@
                           :submitted "%1 lähetti hakemuksen käsiteltäväksi."
                           :unknown "Tuntematon"}
                  :id "Tunniste"
-                 :intro [:div.intro [:p "Täyttämällä ja lähettämällä tämän hakemuksen kaikki hakijat voivat saada oikeuden haettuihin resursseihin. Kunkin hakijan on lisäksi hyväksyttävä käyttöehdot viimeistään hakemuksen käsittelyn jälkeen."]]
+                 :intro [:div.intro [:p "Täytä hakemus ja lähetä se käsiteltäväksi. Hakemukseen voi kutsua jäseniksi useampia hakijoita, ja jokaisen on hyväksyttävä käyttöehdot viimeistään kun hakemus on hyväksytty."]]
```

Comment:
I think there should be a comma before the last "kun" word.

Reply:
Yes, there should

Reply:
I don't think there should. "Viimeistään kun" is an expression that shouldn't be broken in two. A comma is used to separate two clauses that could also exist separately. The kun clause meets this criterion but the one ending with "viimeistään" doesn't.

Reply:
http://www.kielitoimistonohjepankki.fi/ohje/96